test/index.test.js

```diff
@@ -79,10 +79,9 @@ describe('Ramen', function() {
       });
 
       it('should call Ramen.go once', function() {
-        analytics.user().id('12345');
-        analytics.user().traits({ email: 'ryan@ramen.is' });
+        analytics.identify('12345', { email: 'ryan@ramen.is' });
         analytics.page();
-        analytics.called(window.Ramen.go);
+        analytics.calledTwice(window.Ramen.go);
```

Comment:
Why are we changing this here? This would not make sense with the description of the title :/